Sources/MapboxMaps/Location/InterpolatedLocation.swift

```diff
@@ -30,4 +30,12 @@ internal struct InterpolatedLocation: Equatable {
         self.heading = location.headingDirection
         self.accuracyAuthorization = location.accuracyAuthorization
     }
+
+    internal func toLocation() -> Location {
```

Comment:
ah somehow I missed this comment, I do prefer computed properties over function for simple calculation like this, but since I was going to name it with prefix `to` so I used function here. I could change it to a computed property.

Reply:
too late =)) I anyway prefer computed property, just something I overlooked and forgot about it until you pointed it out :-D